docs/mddocs/Overview/FAQ/faq.md

```diff
@@ -88,4 +88,13 @@ You may encounter this error during finetuning, expecially when run 70B model. P
 
 ### `RuntimeError: could not create a primitive` on Windows
 
-This error may happen when multiple GPUs exists for Windows Users. To solve this error, you can open Device Manager (search "Device Manager" in your start menu). Then click the "Display adapter" option, and disable all the GPU device you do not want to use. Restart your computer and try again. IPEX-LLM should work fine this time.
\ No newline at end of file
+This error may happen when multiple GPUs exists for Windows Users. To solve this error, you can open Device Manager (search "Device Manager" in your start menu). Then click the "Display adapter" option, and disable all the GPU device you do not want to use. Restart your computer and try again. IPEX-LLM should work fine this time.
+
+### The first time to run model on Meteor Lake's iGPU(Intel Core Ultra integrated GPU) will takes 5-10 minutes.
+This is caused by intel-extension-for-pytorch's compilation. For windows user, you can install this [intel-extension-for-pytorch](https://github.com/Nuullll/intel-extension-for-pytorch/releases) with MTL [AOT](https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2023-0/ahead-of-time-compilation.html).
```

Comment:
We also have a "MTL" here :)
![image](https://github.com/user-attachments/assets/3c32b8fe-e6d7-4a17-8ab4-a7e0800191c4)